hipchat.go

```diff
@@ -201,9 +204,14 @@ func (c *Client) listen() {
 				continue
 			}
 
+			// parse time
+			time_pieces := strings.Split(attr["ts"], ".")
+			sec, _ := strconv.ParseInt(time_pieces[0], 10, 64)
```

Comment:
completely fair questions. the likelihood is next to nil. the error handling is just best practice, but hipchat communicates with a tweaked version of xmpp. there are some hipchat specific attributes added to the xmpp stanza so having seen that its better to code defensively.